proxysql_exporter.go

```diff
@@ -37,10 +37,11 @@ var (
 	listenAddressF = flag.String("web.listen-address", ":42004", "Address to listen on for web interface and telemetry.")
 	telemetryPathF = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
 
-	mysqlStatusF         = flag.Bool("collect.mysql_status", true, "Collect from stats_mysql_global (SHOW MYSQL STATUS).")
-	mysqlConnectionPoolF = flag.Bool("collect.mysql_connection_pool", true, "Collect from stats_mysql_connection_pool.")
-	mysqlConnectionListF = flag.Bool("collect.mysql_connection_list", true, "Collect connection list from stats_mysql_processlist.")
+	mysqlStatusF                 = flag.Bool("collect.mysql_status", true, "Collect from stats_mysql_global (SHOW MYSQL STATUS).")
+	mysqlConnectionPoolF         = flag.Bool("collect.mysql_connection_pool", true, "Collect from stats_mysql_connection_pool.")
+	mysqlConnectionListF         = flag.Bool("collect.mysql_connection_list", true, "Collect connection list from stats_mysql_processlist.")
 	mysqlDetailedConnectionListF = flag.Bool("collect.detailed.stats_mysql_processlist", false, "Collect detailed connection list from stats_mysql_processlist.")
+	memoryMetricsF               = flag.Bool("collect.stats_memory_metrics", true, "Collect memory metrics from stats_memory_metrics.")
```

Comment:
@AlekSi should this one also be made false?

Reply:
Yes, please.